### PR TITLE
Get full count

### PR DESCRIPTION
**Pull request by JamesJeffryes**
This can be a pretty useful option for queries using pagination. I made it an optional flag because I don't know how badly it will affect performance for some of the existing graph queries

- [X] I updated the README.md docs to reflect this change.
- [X] This is either not a breaking API change, or I incremented the API version.

### CODE REVIEW

File: src/relation_engine_server/utils/arango_client.py [hunk 1]
```diff
@@ -38,6 +38,8 @@ def run_query(query_text=None, cursor_id=None, bind_vars=None, batch_size=100):
         method = 'POST'
         req_json['count'] = True
         req_json['query'] = query_text
+        if full_count:
+            req_json['options'] = {'fullCount': True}
```
Comment (jayrbolton) [inline src/relation_engine_server/utils/arango_client.py]: Interesting.. I hadn't seen that feature in Arango. Should be really useful if we can keep the queries efficient.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
 
 - [ ] I updated the README.md docs to reflect this change.
-- [ ] This is either not a breaking API change, or I incremented the API version.
+- [ ] This is not a breaking API change OR 
+- [ ] This is a breaking API change and I have incremented the API version.
```
[the remark this answers is not in the document]
Comment (JamesJeffryes) [reply .github/pull_request_template.md]: But if you just always check all the boxes, it becomes really easy to do without thinking and then where would we be?

File: src/relation_engine_server/utils/arango_client.py (hunk 1 above)
Comment (JamesJeffryes) [reply src/relation_engine_server/utils/arango_client.py]: BTW the the really low cyclomatic complicity limit drives me nuts on this repo 🤦‍♂️

Comment (jayrbolton) [reply src/relation_engine_server/utils/arango_client.py]: Feel free to remove it or increase it if you find it to be a pain.

To me it is a great prompt to try to see if I can write code without new execution paths, or at the very least split up the function. For example, in this case you could do `req_json['options'] = {'fullCount': full_count}`. Maybe not the most important example.

**Pull request merged**